docs/ja/user-guide/configuration/workflow.md

```diff
@@ -234,6 +236,54 @@ jobs:
 
 サンプルリポジトリ: https://github.com/screwdriver-cd-test/workflow-remote-requires-example
 
+## リモートジョイン
+リモートジョインジョブを作成することもできます。この機能がサポートされているかは、クラスタ管理者に確認してください。
+
+#### 例
+以下の例では、このパイプライン3は、パイプライン2の internal_fork、external_fork とパイプライン4の external_fork が成功した場合に `join_job` を開始します。
+
+![Remote join](../../user-guide/assets/remote-join.png)
+
+Pipeline 3 screwdriver.yaml:
+
+```
+shared:
+  image: node:12
+  steps:
+    - echo: echo hi
+jobs:
+  main:
+    requires: [~commit, ~pr]
+  internal_fork:
+    requires: [main]
+  join_job:
+    requires: [internal_fork, sd@2:external_fork, sd@4:external_fork]
+```
+
+Pipeline 2 screwdriver.yaml:
+
+```
+shared:
+  image: node:12
+  steps:
+    - echo: echo hi
+jobs:
+  external_fork:
+    requires: [~sd@3:main]
+```
+
+Pipeline 4 screwdriver.yaml:
+
+```
+shared:
+  image: node:12
+  steps:
+    - echo: echo hi
+jobs:
+  external_fork:
+    requires: [~sd@3:main]
+```
+
```

Comment:
You have forgotten this.
https://github.com/screwdriver-cd/guide/compare/v0.0.244...v0.0.251#diff-544dc36663b886382be465b2b6297876R268-R274